tsconfig.json

```diff
@@ -2,20 +2,22 @@
   "compilerOptions": {
     "target": "es5",
     "lib": ["dom", "dom.iterable", "esnext"],
-    "allowJs": true,
-    "skipLibCheck": true,
+    "types": ["vite/client", "node"],
+    "allowJs": false,
+    "skipLibCheck": false,
```

Comment:
What does change the `skipLibCheck` flag ?

Reply:
It's a rest of tests

Reply:
But I think it's preferable to false. See [documentation](https://www.typescriptlang.org/tsconfig#skipLibCheck)